firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -377,25 +377,24 @@ public void updateToLatestRelease_whenSignedIn_doesNotCallSignInTester() {
     firebaseAppDistribution.onActivityResumed(activity);
     when(mockCheckForUpdateClient.checkForUpdate())
         .thenReturn(Tasks.forResult(TEST_RELEASE_NEWER_AAB_INTERNAL));
-    Mockito.doReturn(true).when(signInStorage).getSignInStatus();
+    when(mockSignInStorage.getSignInStatus()).thenReturn(true);
     firebaseAppDistribution.updateToLatestRelease();
     verify(mockTesterSignInClient, never()).signInTester(any());
   }
 
   @Test
   public void isTesterSignedIn_afterSuccessfulSignIn_returnsTrue() {
```

Comment:
After the update to these tests I think we should change the names of the tests. We're testing the behavior of the signIn method to make sure it interacts with the storage class. so maybe something like signInTester_afterSuccessfulSignIn_setsSignInStatus?